pipeline/src/test_util.rs

```diff
@@ -9,6 +10,7 @@ use std::sync::Arc;
 use std::{fs::File, io::BufWriter};
```

Comment:
Oh, I think the `#[cfg(feature = "halo2")]` needs to be removed, otherwise you can't compile without the Halo2 features?

Reply:
ah right

Reply:
I think this is actually a mistake I overlooked in https://github.com/powdr-labs/powdr/pull/1161 . We do need to have the feature in Pipeline to forward it to `backend`

Reply:
ah wait the feature is still there so it should be fine, got what you meant now